src/pages/LeeSaemi/Join/Join.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import './Join.scss';
+
+const Join = () => {
+  const [inputId, setInputId] = useState('');
+  const [inputPwd, setInputPwd] = useState('');
+
+  const saveNewUserId = event => {
+    setInputId(event.target.value);
+  };
+
+  const saveNewUserPwd = event => {
+    setInputPwd(event.target.value);
+  };
```

Comment:
```suggestion
  const saveNewUserId = event => {
  const { value } = e.target
    setInputId(event.target.value);
  };
```
처럼 구조분해할당해서 사용할  수 있습니다